CONTRIBUTING.md

```diff
@@ -65,29 +65,14 @@ You are welcome to contribute code to Fundamental Library for Angular in order t
+1.  You must be aware of the Apache License (which describes contributions) and **accept the Developer Certificate of Origin**. This is common practice in major Open Source projects. To make this process as simple as possible, we are using *[CLA assistant](https://cla-assistant.io/)* for individual contributions. CLA assistant is an open source tool that integrates with GitHub very well and enables a one-click experience for accepting the CLA. For company contributors, special rules apply. See the respective section below for details.
 2.  You must follow **code style, quality, and product standards requirements**. You can find more information [HERE](https://github.com/SAP/fundamental-ngx/wiki/Code-Contribution-Guideline).
 
 ### Contributor License Agreement
 
-When you contribute (code, documentation, or anything else), you must be aware that your contribution is covered by the same [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0) that is applied to Fundamental Library for Angular itself.
-In particular, you need to agree to the Individual Contributor License Agreement,
-which can be [found here](https://gist.github.com/CLAassistant/bd1ea8ec8aa0357414e8).
-(This applies to all contributors, including those contributing on behalf of a company). If you agree to its content, you simply have to click on the link posted by the CLA assistant as a comment to the pull request. Click it to check the CLA and, if you agree to it, accept it on the following screen. CLA assistant saves this decision for upcoming contributions and notifies you if there is any change to the CLA in the meantime.
-
-#### Company Contributors
-
-If employees of a company contribute code, in **addition** to the individual agreement above, there needs to be one company agreement submitted. This is mainly for the protection of the contributing employees.
-
-A company representative authorized to do so needs to download, fill out, and print
-the [Corporate Contributor License Agreement](/docs/SAP%20Corporate%20Contributor%20License%20Agreement.pdf) form. Then either:
-
--   Scan it and e-mail it to [opensource@sap.com](mailto:opensource@sap.com),
--   Fax it to: +49 6227 78-45813,
--   Send it by mail to: _Industry Standards & Open Source Team, Dietmar-Hopp-Allee 16, 69190 Walldorf, Germany_.
-
-The form lists all employees who are authorized to contribute on behalf of your company. When this list changes, please let us know.
+## Developer Certificate of Origin (DCO)
 
+Due to legal reasons, contributors will be asked to accept a DCO before they submit the first pull request to this projects, this happens in an automated fashion during the submission process. SAP uses [the standard DCO text of the Linux Foundation](https://developercertificate.org/).
```

Comment:
before they submit **their** first pull request*

Reply:
to this project* (singular)

Reply:
"Due to legal reasons, contributors will be asked to accept a DCO before they submit their first pull request to this project. This process will happen in an automated fashion during a submission."